src/packages/end-to-end/src/__tests__/api/auth/acl/access-control-list.test.ts

```diff
@@ -21,12 +21,12 @@ const user = new UserProfile({
 	displayName: 'Test User',
 });
 
-const artistDataProvider = new BaseDataProvider<any, Artist>('artist');
+const artistDataProvider = new BaseDataProvider<any>('artist');
 
 @Entity('Artist', {
 	provider: artistDataProvider,
 })
-export class Artist extends GraphQLEntity<any> {
+export class Artist {
 	public dataEntity!: any;
```

Comment:
Ok, there are a few so we can just search for any `public dataEntity!` and remove.